synfig-core/src/tool/optionsprocessor.cpp

```diff
@@ -684,6 +696,22 @@ Job SynfigCommandLineParser::extract_job()
 	    throw SynfigToolException(SYNFIGTOOL_MISSINGARGUMENT,
                                   _("No input file provided."));
 	}
+        
+	if(!set_renderer.empty())
+	{
+		auto renderers = rendering::Renderer::get_renderers();
+		auto ri = renderers.find(set_renderer);
+		if (ri == renderers.end() || !ri->second)
+		{
+			synfig::error("Invalid renderer: %s", set_renderer.c_str()); 
```

Comment:
@veermetri05  let this message translatable: surround the string with `_("my translatable string")`.

@ice0 do you think it should be std::cerr or synfig::error?

Reply:
I think it's ok to use `synfig::error` here.